crates/ethereum-forks/src/hardfork/optimism.rs

```diff
@@ -0,0 +1,25 @@
+use crate::{hardfork, Hardfork};
+use core::{
+    fmt,
+    fmt::{Display, Formatter},
+    str::FromStr,
+};
+
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+hardfork!(
```

Comment:
this should document that while this only defines the optimism hardfork variants it's still expected that for usage in `Hardforks` they should be mixed with ehereumhardforks